source/org/zfin/expression/repository/HibernateExpressionRepository.java

```diff
@@ -1803,7 +1803,7 @@ public List<ExpressionResult> getNonEfgExpressionResultsByFish(Fish fish) {
                 "        and fishox = xpExp.fishExperiment " +
                 "        and xpRslt.expressionExperiment = xpExp " +
                 "        and xpExp.gene != null" +
-                "        and xpExp.gene.zdbID not like :markerId)";
+                "        and xpExp.gene.zdbID not like :markerId";

```

Comment:
I don't understand how this ever worked in the past?

Reply:
Good find.